### ISSUE TITLE
Illustrator Layers, Paintbrush Tool

### ISSUE THREAD

**Issue by flickitty** [bodymovin/bodymovin#164]
Discovered an issue with Illustrator Layers, even after converting to vector using "Create Shapes from Vector Layer". It seems the problem stems from using the Paintbrush Tool in illustrator. The problem still manifests after converting strokes to paths (in Illustrator).

The solids and alpha become inverted.

Once I retraced the shape with the Pen Tool and added a stroke, everything exported just fine, and BodyMovin properly exported the shape as SVG. It also works just fine if I use Image Trace in Illustrator to create the vector shapes.

In short- don't use the Illustrator Paintbrush Tool, I spent many wasted hours trying to fix this.

**Comment by bodymovin** [comment]
Could you send me an example of the Paintbrush that doesn't work?
I tried with a simple one and AE created a fill when converted to shape.
